### PR TITLE
Aditya#basic models

### PR DESCRIPTION
**Pull request by itaditya**

### CODE REVIEW

File: models/Arena.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+const mongoose = require('mongoose');
+const {
+  Schema
+} = mongoose;
+const MediaSchema = require('./schema/Media');
+const ArenaSchema = new Schema({
+  _event: {
+    type: Schema.ObjectId,
```
Comment (ankitjain28may) [inline models/Arena.js:8]: rather typing again and again, we can define ObjectId at the top
`const ObjectId = Schema.ObjectId;`

**Review by ankitjain28may — COMMENTED** [review]
Looks Good To Me

File: models/Arena.js (hunk 1 above)
Comment (itaditya) [reply models/Arena.js:8]: Nice idea, even better we can use the new ES6 destructuring feature 
`const { ObjectId } = Schema`

Comment (itaditya) [reply models/Arena.js:8]: Done in #22

Comment (ankitjain28may) [reply models/Arena.js:8]: Great

**Pull request merged**